calico_containers/calicoctl.py

```diff
@@ -57,12 +57,12 @@
 from netaddr.core import AddrFormatError
 from prettytable import PrettyTable
 
-from calico_containers.adapter.datastore import (ETCD_AUTHORITY_ENV,
```

Comment:
Can calicoctl.py still be invoked from the command line? Does it require any fiddly pythonpath stuff?